src/redux-toolkit/features/authSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  authStatus: localStorage.getItem("token") ? true : false,
+  token: localStorage.getItem("token"),
```

Comment:
you could use just 
`const initialState = {
    token: localStorage.getItem("token") || null
}`
and when you want to use auth status you could just substitute it with the token variable

Reply:
tried it. but using authStatus to toggle between login and logout button in the navbar component and null !== false, so toggling was not working.

Reply:
alright then that's ok.